tests/framework/installer/ceph_manifests.go

```diff
@@ -2077,7 +2216,7 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: ` + storageClassName + `
-provisioner: ` + SystemNamespace(namespace) + `.cephfs.csi.ceph.com
+provisioner: ` + systemNamespace + `.cephfs.csi.ceph.com
```

Comment:
@travisn it was needed to change this one as we moved the CSI snapshot and clone tests to helm tests because for helm tests both the system namespace and the name remain same. so with the earlier change, the tests are failing, and also new modifications are done on other places also to pass the system namespace can we retrigger the complete test suite on this PR?

Reply:
sample failed test results are here https://jenkins.rook.io/blue/rest/organizations/jenkins/pipelines/rook/pipelines/rook/branches/PR-6066/runs/16/nodes/55/steps/125/log/?start=0

Reply:
The tests are already going again from your last update, let's see how it goes this time.